src/PlaidLink.js

```diff
@@ -51,6 +51,14 @@ class PlaidLink extends Component {
     // The public_key associated with your account; available from
     // the Plaid dashboard (https://dashboard.plaid.com)
     publicKey: PropTypes.string.isRequired,
+    
+    // The country code allows you to select your bank of choice from
+    // your country of choosing (You can add you own country codes)
+    countryCodes: PropTypes.arrayOf(
+      PropTypes.oneOf([
+        'CA',
```

Comment:
Should this also include the following countries codes?

CA, FR, ES, GB, US?

According to the docs those countries have at least some support and CA, GB and US have full support.

Reply:
Yep! Sorry when I was writing it I only wanted to include Canadian banks. But yes you can add countries by their ISO 3166-1 code